### ISSUE TITLE
Text value by default on build (initialValue)

### ISSUE THREAD

**Issue by tgbarker** [adar2378/pin_code_fields#153]
I am trying to create a PinCodeTextField with a visible default value in a Stateless Widget.

```
PinCodeTextField(
        appContext: context,
        length: 4,
        onChanged: (String value) {},
        enabled: false,
        controller: TextEditingController(text: "1234"));
```

This just shows 4 blank grey squares and does not show the value assigned in "1234".

I find this use reasonable as at one point I am asking the user for a code, and another point showing this code, so as to display the same style/UI I would like to use the same widget.

If enabled is true then when I click on any of the squares, they are filled with the value of my controller.

Am I doing something wrong and/or is it the following?

The widget is based on TextFormField, there is no initialValue set at about line 669 of pin_code_fields.dart. Thus it would make sense to attribute an initialValue from the given TextEditingController in the construction of the PinCodeTextField.

Thanks in advance.

**Comment by adar2378** [comment]
I'll check from my side and find a solution to this soon

**Comment by tgbarker** [comment]
> 
> 
> I'll check from my side and find a solution to this soon

Thanks. In fact, what I suggested will not solve the problem but should probably be done anyways, as your TextFormField is hidden. It is in fact _inputList that is not filled with the default value on creation and should probably be taken from the TextEditController passed on creation of the widget. Thanks.

**Comment by JHBitencourt** [comment]
Since this was only resolved on `7.0.0` with null safety support, and I don't plan on upgrading my app right now to Flutter 2.0, there is any workaround that can be done on version `6.1.0`?
